voltixApp/VoltixApp/Model/Solana/SolanaRecentBlockhash.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension SolanaService {
+	
+	class SolanaRecentBlockhashResponse: Codable {
```

Comment:
I don't understand why these classes are under SolanaService extension. can you explain?